test/analysis_ee_ZH_cfg_pdebug.py

```diff
@@ -0,0 +1,194 @@
+'''Example configuration file for an ee->ZH->mumubb analysis in heppy, with the FCC-ee
```

Comment:
no need for specific cfg? 
all of our test cfgs: analysis_ee_ZH, guns, etc, could be put in pdebug mode just by changing one line. 
So no need to duplicate config